app/views/candidate_interface/course_choices/options_for_course.html.erb

```diff
@@ -6,11 +6,17 @@
     <%= form_with model: Course.new, url: candidate_interface_course_choices_course_path(provider_code: params[:provider_code]), builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
       <%= f.govuk_error_summary %>
 
-      <h1 class="govuk-heading-xl">
-        <%= t('page_titles.which_course') %>
-      </h1>
+      <%= f.govuk_radio_buttons_fieldset :code, legend: { size: 'xl', text: t('page_titles.which_course') } do %>
+        <div class="govuk-!-margin-top-6">
+          <% @courses.map do |course| %>
+            <%= f.govuk_radio_button :code, course.code, label: { text: "#{course.name} (#{course.code})", size: 'm' } %>
```

Comment:
All radio options should look the same (normal weight, standard size). I think removing the `size` attribute should fix this.